cmd/hypper/hypper.go

```diff
@@ -23,7 +23,7 @@ func main() {
 	logger := logcli.NewStandard()
 	log.Current = logger
 
-	actionConfig := new(helmAction.Configuration)
+	actionConfig := new(action.Configuration)
```

Comment:
Unit tests pass, but running `hypper list` fails with an invalid mem address or nil pointer. I suppose we need to initialize our `hypper/pkg/action.Configuration` correctly here. Still, I'm looking at how to.

Reply:
fixed in https://github.com/rancher-sandbox/hypper/pull/46/commits/c7fd3ac77bdd38273cf9dec3ff1c26b6ad9e5305.